nncf/common/graph/utils.py

```diff
@@ -114,3 +115,17 @@ def get_number_of_quantized_ops(
             else:
                 nodes_to_see.extend(graph.get_next_nodes(node))
     return len(quantized_ops)
+
+
+def get_reduction_axes(axes_to_keep: Union[List[int], ReductionAxes], shape_to_reduce: List[int]) -> ReductionAxes:
+    """
+    Returns reduction axes without axes needed to keep.
```

Comment:
Please rephrase

Reply:
pleasae suggest your variant

Reply:
I suggest to keep function name and parameters  of the original openvino node utils function